### PR TITLE
Remove non-exist UI element type for 128x64 and text

### PR DESCRIPTION
**Pull request by howard0su**

### CODE REVIEW

File: src/gui/320x240x16/_button.c [hunk 1]
```diff
@@ -13,6 +13,26 @@
...
  */
 
+const struct ImageMap image_map[] = {
+    {"media/btn96_24" IMG_EXT, 96, 24, 0, 0}, /*FILE_BTN96_24 */
+    {"media/btn48_24" IMG_EXT, 48, 24, 0, 0}, /*FILE_BTN48_24 */
+    {"media/btn96_16" IMG_EXT, 96, 16, 0, 0}, /*FILE_BTN96_16 */
+    {"media/btn64_16" IMG_EXT, 64, 16, 0, 0}, /*FILE_BTN64_16 */
+    {"media/btn48_16" IMG_EXT, 48, 16, 0, 0}, /*FILE_BTN48_16 */
+    {"media/btn32_16" IMG_EXT, 32, 16, 0, 0}, /*FILE_BTN32_16 */
+    {"media/spin192"  IMG_EXT,192, 16, 0, 0}, /*FILE_SPIN192 */
```
Comment (PhracturedBlue) [inline src/gui/320x240x16/_button.c]: Lint Error:
```
23:  Missing space after ,  
```

File: src/gui/gui.h [hunk 2]
```diff
@@ -95,7 +95,7 @@ struct ImageMap {
 
 #include "buttons.h"
 
-extern const struct ImageMap image_map[IMAGE_MAP_END];
+//extern const struct ImageMap image_map[IMAGE_MAP_END];
```
Comment (PhracturedBlue) [inline src/gui/gui.h]: Lint Error:
```
98:  Should have a space between // and comment  
```

File: src/gui/128x64x1/_gui.h [hunk 3]
```diff
@@ -1,41 +1,17 @@
...
-    BUTTON_48x16,
-    BUTTON_32x16,
+    BUTTON_GENERIC
 };
 
-#define DIALOG_BUTTON BUTTON_DEVO10
+#define DIALOG_BUTTON BUTTON_GENERIC
 
 #define ARROW_LEFT  (&image_map[DRAW_BTN32_15])
 #define ARROW_RIGHT (&image_map[DRAW_BTN32_15])
```
Comment (PhracturedBlue) [inline src/gui/128x64x1/_gui.h]: these should be removed, right?

File: src/gui/text/_gui.h [hunk 4]
```diff
@@ -1,39 +1,14 @@
...
-    BUTTON_48x16,
-    BUTTON_32x16,
+    BUTTON_GENERIC
 };
 
-#define DIALOG_BUTTON BUTTON_DEVO10
+#define DIALOG_BUTTON BUTTON_GENERIC
 
 #define ARROW_LEFT  (&image_map[DRAW_BTN32_15])
 #define ARROW_RIGHT (&image_map[DRAW_BTN32_15])
```
Comment (PhracturedBlue) [inline src/gui/text/_gui.h]: these also should be removed?

File: src/gui/textsel.c [hunk 5]
```diff
@@ -281,6 +280,9 @@ void GUI_TextSelectEnablePress(guiTextSelect_t *select, u8 enable)
         }
         return;
     }
+
+#if LCD_DEPTH != 1
+    guiObject_t *obj = (guiObject_t *)select;
```
Comment (PhracturedBlue) [inline src/gui/textsel.c]: i prefer not to add '#if' unless absolutely necessary.  better to define 'select' at the top of the function and wrap this in a if(LCD_DEPTH != 1) {}

File: src/gui/128x64x1/_gui.h (hunk 3 above)
Comment (howard0su) [reply src/gui/128x64x1/_gui.h]: removed

File: src/gui/text/_gui.h (hunk 4 above)
Comment (howard0su) [reply src/gui/text/_gui.h]: removed

File: src/gui/textsel.c (hunk 5 above)
Comment (howard0su) [reply src/gui/textsel.c]: sure.

**Comment by howard0su** [comment]
@PhracturedBlue check again.

**Pull request merged**